### ISSUE TITLE
Spurious error when using closed `TypedDict` with `__extra_items__`

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#7784]
When importing `TypedDict` from `typing_extensions`, pyright generates a false positive error in the following code:

```python
from typing_extensions import TypedDict

class Movie(TypedDict, closed=True):
    name: str
    __extra_items__: bool  # Error reported here
```

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.361, which I just published.
